spec.html

```diff
@@ -34990,8 +34992,80 @@ <h2>Syntax</h2>
           [+UnicodeMode] `-`
           CharacterClassEscape[?UnicodeMode]
           CharacterEscape[?UnicodeMode]
+
+        ClassSetExpression ::
+          ClassUnion
+          ClassIntersection
+          ClassSubtraction
+
+        ClassUnion ::
+          ClassSetRange ClassUnion?
+          ClassSetOperand ClassUnion?
+
+        ClassIntersection ::
+          ClassSetOperand `&amp;&amp;` [lookahead != `&amp;`] ClassSetOperand
+          ClassIntersection `&amp;&amp;` [lookahead != `&amp;`] ClassSetOperand
+
+        ClassSubtraction ::
+          ClassSetOperand `--` ClassSetOperand
+          ClassSubtraction `--` ClassSetOperand
+
+        ClassSetRange ::
+          ClassSetCharacter `-` ClassSetCharacter
+
+        ClassSetOperand ::
+          ClassSetCharacter
+          ClassStringDisjunction
+          NestedClass
```

Comment:
```suggestion
          NestedClass
          ClassStringDisjunction
          ClassSetCharacter
```
... so that the order of the right-hand sides matches the order in which those nonterminals are defined. (Like `ClassSetExpression` does, for example.)

Reply:
Applied!